src/daft-sql/src/lib.rs

```diff
@@ -147,33 +147,11 @@ mod tests {
     #[case::join_with_filter("select * from tbl2 join tbl3 on tbl2.id = tbl3.id and tbl2.val > 0")]
     #[case::from("select tbl2.text from tbl2")]
     #[case::using("select tbl2.text from tbl2 join tbl3 using (id)")]
-    #[case(
-        r"
-    select
-        abs(i32) as abs,
-        ceil(i32) as ceil,
-        floor(i32) as floor,
-        sign(i32) as sign
-    from tbl1"
-    )]
-    #[case("select round(i32, 1) from tbl1")]
-    #[case::clip_int_to_float("select clip(i32, 1.5, 2.5) from tbl1")]
-    #[case::clip_float_to_int("select clip(f32, 1, 2) from tbl1")]
-    #[case::clip_null_lower("select clip(i32, NULL, 2) from tbl1")]
-    #[case::clip_null_upper("select clip(i32, 1, NULL) from tbl1")]
-    #[case::clip_float_null_lower("select clip(f32, NULL, 2.5) from tbl1")]
-    #[case::clip_float_null_upper("select clip(f32, 1.5, NULL) from tbl1")]
-    #[case::clip_float_lower_int_upper("select clip(i32, 1.5, 2) from tbl1")]
-    #[case::clip_int_lower_float_upper("select clip(i32, 1, 2.5) from tbl1")]
-    #[case::clip_float_lower_int_upper_float_col("select clip(f32, 1.5, 2) from tbl1")]
-    #[case::clip_int_lower_float_upper_float_col("select clip(f32, 1, 2.5) from tbl1")]
-    #[case::groupby("select max(i32) from tbl1 group by utf8")]
     #[case::orderby("select * from tbl1 order by i32")]
     #[case::orderby("select * from tbl1 order by i32 desc")]
     #[case::orderby("select * from tbl1 order by i32 asc")]
     #[case::orderby_multi("select * from tbl1 order by i32 desc, f32 asc")]
     #[case::whenthen("select case when i32 = 1 then 'a' else 'b' end from tbl1")]
-    #[case::globalagg("select max(i32) from tbl1")]
```

Comment:
Why do we no longer support these (and the other tests below)?

Reply:
so we do support them, but in daft-sql, the functions are not yet registered until `daft/lib.rs`. We have duplicate tests for many of these in python and sql anyways. If you think it's important to keep these, we could register the functions for `daft-sql` during testing, but I don't think these are overly important to keep.

Reply:
Sounds good. I think we should probably move all of our sql tests to sqllogictest format once we get that up and running anyway.